lib/zstd.h

```diff
@@ -2067,6 +2070,13 @@ ZSTDLIB_STATIC_API size_t ZSTD_CCtx_refPrefix_advanced(ZSTD_CCtx* cctx, const vo
  * documentation (below) before setting this parameter. */
 #define ZSTD_c_enableMatchFinderFallback ZSTD_c_experimentalParam17
 
+/*  ZSTD_c_maxBlockSize
+ *
+ *  Default is ZSTD_BLOCKSIZE_MAX.
+ *
```

Comment:
Please add a little more detail here. Say what this parameter controls, what the min/max are (and why), and explain what setting to 0 means.